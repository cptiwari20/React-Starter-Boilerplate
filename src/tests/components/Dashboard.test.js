import React from 'react';
import { shallow } from 'enzyme';
import Dashboard from '../../components/Dashboard';

it('should correctly render Dashboard', () => {
  const wrapper = shallow(<Dashboard />)
  expect(wrapper).toMatchSnapshot();
})
