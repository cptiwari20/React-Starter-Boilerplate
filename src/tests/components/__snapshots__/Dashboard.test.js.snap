// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Dashboard 1`] = `
<div>
  Dashboard
</div>
`;
